components/AppBar/BUILD

```diff
@@ -119,28 +121,19 @@ mdc_examples_swift_library(
     ],
 )
 
-mdc_objc_library(
+mdc_unit_test_objc_library(
```

Comment:
Why remove the private visibility for these 2?

Reply:
It's not - the new macros have the visibility defaulting to private. We would only override visibility on these targets if it was anything except private.

```
def mdc_unit_test_objc_library(
    name,
    extra_srcs = [],
    deps = [],
    sdk_frameworks = [],
    visibility = ["//visibility:private"],
    testonly = 1,
    **kwargs):
    """Declare an mdc_objc_library for unit test sources."""
    mdc_objc_library(
```